.github/workflows/build_and_push_performance_test.yml

```diff
@@ -74,5 +74,5 @@ jobs:
       - name: Notify Slack channel if this job failed
         if: ${{ failure() }}
         run: |
-          json='{"text":"<!here> CI is failing in <https://github.com/cds-snc/notification-api/actions/runs/${GITHUB_RUN_ID}|notification-api> !"}'
+          json='{"text":"<!here> CI is failing in <https://github.com/cds-snc/notification-api/actions/runs/'${{ github.run_id }}'|notification-api> !"}'
```

Comment:
🤔 Did you test this? there's a lot of single quotes in here!

Reply:
Yes! Well, i went through many iterations in admin to get this right.  Here is the equivalent code in admin that I confirmed posted the message correctly to slack: https://github.com/cds-snc/notification-admin/pull/2041/files